### ISSUE TITLE
Private SSH key not deleted in Ubuntu VM

### ISSUE THREAD

**Issue by ksoderstrom** [bitrise-io/steps-git-clone#12]
When building an android app with the Ubuntu image, the private SSH key is not deleted in the git-clone steps, since it's trying to use the `-P` parameter when calling `rm`. This parameter is available in the BSD `rm` used on Mac OS X, but not the GNU version used on Ubuntu.

```
 (i) Removing private key file: /root/.ssh/bitrise
rm: invalid option -- 'P'
Try 'rm --help' for more information.
```

**Comment by viktorbenei** [comment]
good catch, added to our backlog! 😉

**Comment by viktorbenei** [comment]
Fixed in 3.3.0 - thanks for reporting!
